py4DSTEM/process/diskdetection/diskdetection_aiml_cuda.py

```diff
@@ -219,7 +219,8 @@ def _find_Bragg_disks_aiml_single_DP_CUDA(DP, probe,
                                   peaks = None,
                                   get_maximal_points = None,
                                   blocks = None,
-                                  threads = None):
+                                  threads = None,
+                                  model_path="path/to/model"):
```

Comment:
not sure if "path/to/model" makes sense as a default?

Reply:
@alex-rakowski  can you recommend a better solution here? `model_path` was being used but never defined, so I just added it to the argument list with a nonsense default as a stopgap to remove the syntax error

Reply:
I just changed it to `None` - this will have the correct behavior when passed to _get_latest_model from diskdetection_aiml.py